lib/classes/Variables.js

```diff
@@ -822,6 +827,22 @@ class Variables {
       });
   }
 
+  getValueStrToBool(variableString) {
+    return BbPromise.try(() => {
+      const groups = variableString.match(this.strToBoolRefSyntax);
```

Comment:
I believe we need to take into account that `variableString` may not be string (e.g. internally in other places we already do that: https://github.com/serverless/serverless/blob/56f3c56388faecadc81cf5572e21f98b8b8ae25c/lib/classes/Variables.js#L534 )

And in case it's not a string I would just do `return Boolean(variableString)`.

e.g. I'm pretty sure`${strToBool(strToBool('true'))}` will crash badly if we won't do that (it'll be nice to add test for this case)

Reply:
I found using `.match(this.variableSyntax)` does not work as expected.

Reply:
This fails to match `variableString.match(this.variableSyntax)`.
This fails to match `param.match(this.variableSyntax)`.

Reply:
My point was not about `value.match(this.variableSyntax)` but about `_.isString(value)`

Reply:
Ahh. Thanks for clarifying.